### PR TITLE
Disable ability to add DI configuration from plugins

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

Loading DI configuration from plugins is not working as excpected for now.
We need to rework some parts of code before being able to do it  in the right way.

I did not completely remove the code, as we may want to reuse some parts later.

### CODE REVIEW

File: tests/units/Glpi/Kernel.php [hunk 1]
```diff
@@ -125,6 +125,8 @@ public function getDb() {
 
    /**
     * Test compilation of services configured in a plugin.
+    *
+    * @ignore
```
Comment (trasher) [inline tests/units/Glpi/Kernel.php]: What is the purpose?

**Comment by trasher** [comment]
You have some CS issues, that looks good to me.

File: tests/units/Glpi/Kernel.php (hunk 1 above)
Comment (cedric-anne) [reply tests/units/Glpi/Kernel.php]: Ignoring the test, without having to remove the method.

Comment (trasher) [reply tests/units/Glpi/Kernel.php]: ok :)

**Pull request merged**